src/components/FavBtn/FavBtn.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+import './FavBtn.sass'
+
+const FavBtn = props => {
+  const {id, isFav, handleFavButtonClicked} = props
```

Comment:
Use deconstruction on function's argument level

```suggestion
  const FavBtn = ({ id, isFav, handleFavButtonClicked }) =>
```